internal/appsec/rules_manager.go

```diff
@@ -64,6 +64,17 @@ type (
 	rulesData     struct {
 		RulesData []ruleDataEntry `json:"rules_data"`
 	}
+
+	actionEntry struct {
+		ID         string `json:"id"`
+		Type       string `json:"type"`
+		Parameters struct {
+			StatusCode     int    `json:"status_code"`
+			GRPCStatusCode int    `json:"grpc_status_code,omitempty"`
```

Comment:
Given the fact that the product only considers HTTP actions for now, I think you need to avoid zero Go values on `GRPCStatusCode` by using a pointer which will be nil if the field is not set/sent in the RC payload:
```suggestion
			GRPCStatusCode *int    `json:"grpc_status_code,omitempty"`
```

If you receive `0`, that would be a backend mistake

Reply:
[68a7c53](https://github.com/DataDog/dd-trace-go/pull/2044/commits/68a7c53e5f474bcd23f713fb9b2e295c0759066b)